larva/src/main/java/nl/nn/adapterframework/testtool/queues/IQueue.java

```diff
@@ -0,0 +1,8 @@
+package nl.nn.adapterframework.testtool.queues;
+
+import nl.nn.adapterframework.configuration.ConfigurationException;
+
+public interface IQueue {
+
+	default void configure() throws ConfigurationException {}
```

Comment:
Hier komen dan ook nog wat functionele methods in?

Reply:
Jazeker, het is nog maar een opzet, ik wil eerst alles omzetten en dan kijken wat allemaal nodig/handig is.